cli/src/cmd/forge/build.rs

```diff
@@ -138,6 +138,15 @@ pub struct CoreBuildArgs {
     )]
     #[serde(skip)]
     pub config_path: Option<PathBuf>,
+
+    #[clap(
+        help_heading = "PROJECT OPTIONS",
+        help = r#"How to treat revert (and require) reason strings. Possible values are "default", "strip", "debug" and "verboseDebug""#,
```

Comment:
Maybe this is better? Also I think it should be under "compiler options"

```suggestion
        help = r#"Revert string configuration. Possible values are "default", "strip" (remove), "debug" (Solidity-generated revert strings) and "verboseDebug""#,
```